.npmrc

```diff
@@ -0,0 +1,2 @@
+@yalesites-org:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=${YALESITES_BUILD_TOKEN}
```

Comment:
I don't think this line is needed. While you need a token to update/build packages, I believe public packages are open to everyone. While your instructions work, getting an authToken add unnecessary complexity to contributing to this project. I think this line can be removed.

Reply:
I think it is required. GitHub packages (even public ones) need this in order to be installed. So, while the repo is public, in order to install the package version, you have to have an auth Token... It's annoying, but that's what I found

Reply:
For reference, here's what I've found: https://github.community/t/download-from-github-package-registry-without-authentication/14407/2